.github/workflows/ci.yml

```
@@ -0,0 +1,125 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
```

This is for `2.x` branch only:

```suggestion
      - 2.x
```